tests/pr_test.go

```diff
@@ -77,3 +77,20 @@ func TestRunUpgradeDA(t *testing.T) {
 		assert.NotNil(t, output, "Expected some output")
 	}
 }
+
+func TestRunDAWithObs(t *testing.T) {
```

Comment:
No need to create a new test. Just set `provision_atracker_cos` to true in the current test